presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnector.java

```diff
@@ -25,16 +25,19 @@
 import io.prestosql.spi.connector.ConnectorSplitManager;
```

Comment:
Can you please extract this commit as separate PR so we could merge this without blocking on postgres changes?

Reply:
Done: https://github.com/prestosql/presto/pull/1181